ansible-tests/validations/neutron-sanity-check.yaml

```diff
@@ -8,6 +8,33 @@
         Neutron's configuration.
       groups:
         - post-deployment
+    config_files:
+    - /etc/neutron/neutron.conf
+    - /usr/share/neutron/neutron-dist.conf
+    - /etc/neutron/metadata_agent.ini
+    - /etc/neutron/metering_agent.ini
+    - /etc/neutron/dhcp_agent.ini
+    - /etc/neutron/plugins/ml2/openvswitch_agent.ini
+    - /etc/neutron/conf.d/ml2_conf_cisco.ini
+    - /etc/neutron/conf.d/cisco_cfg_agent.ini
+    - /etc/neutron/conf.d/cisco_router_plugin.ini
+    - /etc/neutron/dhcp_agent.ini
+    config_dirs:
+    - /usr/share/neutron/l3_agent
+    - /etc/neutron/conf.d/common
+    - /etc/neutron/conf.d/neutron-l3-agent
+    - /usr/share/neutron/neutron-lbaas-dist.conf
+    - /etc/neutron/lbaas_agent.ini
+    - /etc/neutron/conf.d/neutron-lbaas-agent
+    - /etc/neutron/conf.d/neutron-lbaasv2-agent
+    - /etc/neutron/conf.d/neutron-metadata-agent
+    - /etc/neutron/conf.d/neutron-metering-agent
+    - /etc/neutron/conf.d/neutron-netns-cleanup
+    - /etc/neutron/conf.d/neutron-openvswitch-agent
+    - /etc/neutron/conf.d/neutron-ovs-cleanup
+    - /etc/neutron/conf.d/neutron-bsn-agent
+    - /etc/neutron/conf.d/neutron-cisco-cfg-agent
+    - /etc/neutron/conf.d/neutron-dhcp-agent
   tasks:
   - name: Run neutron-sanity-check
-    command: neutron-sanity-check --config-file /etc/neutron/neutron.conf
+    script: files/neutron-sanity-check '"{{config_files|join(" ")}}" "{{config_dirs|join(" ")}}"'
```

Comment:
The script expects 2 arguments, you need to remove the single quotes otherwise you'll get an unbound variable $2 error when running the script.